das_client/app/lib/pages/journey/journey_table/widgets/header/next_stop.dart

```diff
@@ -16,13 +16,7 @@ class NextStop extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Row(
-      mainAxisSize: MainAxisSize.min,
-      children: [
-        _tappableNextStopIdentifier(context),
-        Expanded(child: SizedBox(width: double.infinity)),
-      ],
-    );
+    return _tappableNextStopIdentifier(context);
```

Comment:
Nitpick: Inhalt von `_tappableNextStopIdentifier` direkt in Build-Methode. So bringt es m.M. keine Mehrwert mehr.